lib/helper/import.js

```diff
@@ -0,0 +1,100 @@
+'use babel';
+
+const atom = global.atom;
+const Path = require('path');
+const FileSystem = require('fs-plus');
+const Electron = require('electron');
+const Storage = require('./../helper/storage.js');
+const XMLParser = require('xml-js');
+const server_config = require('./../config/server-schema.json');
+const folder_config = require('./../config/folder-schema.json');
+
+class Import {
+  constructor() {
+    const self = this;
+  }
+
+  import(configuration_view) {
+    const self = this;
+    Electron.remote.dialog.showOpenDialog(null, { title: 'Select file to import', defaultPath: Electron.remote.app.getPath("desktop"), buttonLabel: 'Import', filters: [{ name: 'All Files', extensions: ['*'] }, { name: 'remote-ftp config file', extensions: ['ftpconfig'] }, { name: 'FileZilla site export file', extensions: ['xml'] }], properties: ['openFile', 'showHiddenFiles'] }, (filePaths, bookmarks) => {
+      if (filePaths) {
+        filePaths.forEach((filePath, index) => {
+          if (Path.extname(filePath).toLowerCase() == '.xml') {
+            FileSystem.readFile(filePath, function (err, data) {
+              var json = XMLParser.xml2js(data, { compact: true });
+              if (typeof json['FileZilla3'] !== 'undefined') {
+                self.importFileZilla3(json['FileZilla3']['Servers']['Folder'], null);
+                configuration_view.reload();
+              }
+            });
+          } else if (Path.basename(filePath).toLowerCase() == '.ftpconfig') {
+            FileSystem.readFile(filePath, function (err, data) {
+              var json = JSON.parse(data);
+              self.importFtpConfig(json);
+              configuration_view.reload();
+            });
+
+          }
+        });
+      }
+
+    });
+  }
+
+  importFileZilla3(data, parent) {
+    const self = this;
+    if (typeof data['Server'] !== 'undefined') {
+      if (!Array.isArray(data['Server'])) {
+        data['Server'] = [data['Server']];
+      }
+      data['Server'].forEach((server_data, index) => {
+        let newconfig = JSON.parse(JSON.stringify(server_config));
+        newconfig.name = server_data['Name']['_text'];
+        newconfig.host = server_data['Host']['_text'];
+        if (typeof server_data['Port'] !== 'undefined')
+          newconfig.port = server_data['Port']['_text'];
+        if (typeof server_data['User'] !== 'undefined')
+          newconfig.user = server_data['User']['_text'];
+        if (typeof server_data['Pass'] !== 'undefined')
+          newconfig.password = Buffer.from(server_data['Pass']['_text'], server_data['Pass']['_attributes']['encoding']).toString();
```

Comment:
Check for encrypted password in new Filezilla versions and ask for filezilla master pasword to decrypt the password.

Reply:
Password can be encoding as base64, plain or encrypted. 
Here's the encryption information: https://forum.filezilla-project.org/viewtopic.php?f=3&t=64&start=1005#p156191

Reply:
Made import for plain, and for now warning for elements with master password... It doesn't look so easy for crypted one.

Reply:
I think that this is too hard for me... ;) Could You try to implement it?

Reply:
Yes encryption is always very complex and difficult to understand. I will try it :)